src/app/features/courses/courses.component.ts

```diff
@@ -10,16 +10,21 @@ export class CoursesComponent {
   @Input() courses:CourseInfo[]=[];
   
   courseInfo:CourseInfo|undefined=undefined;
+  searchedCourses:CourseInfo[]=[];
+  searched:boolean=false;
```

Comment:
Is this really necessary? It is initialized with `false` value, and it reassigned to `true` on the first search. There is no scenario where it will be `false` again.

Reply:
Thank you! It was just a baby version of the searchBar, but now grown up, refined
Done